modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -405,6 +405,26 @@ public void testAddObjectEntry() throws Exception {
 					"\"listTypeEntryKeyRequired\"",
 				objectEntryValuesException.getMessage());
 		}
+
+		try {
+			_addObjectEntry(
+				HashMapBuilder.<String, Serializable>put(
+					"emailAddress", "redwacky@liferay.com"
+				).put(
+					"firstName", RandomTestUtil.randomString(281)
+				).put(
+					"listTypeEntryKey", "listTypeEntryKey1"
```

Comment:
It would be better to pass just the required ones and "firstName", so we can remove "listTypeEntryKey"

Reply:
Sure!
Thanks for the tip!